assets/css/style.css

```diff
@@ -409,3 +428,125 @@ header {
   font-weight: 500;
   width: 142px;
 }
+
+.nav-ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: none;
+}
+
+.nav-ul > li > a {
+  text-decoration: none;
+  color: #344563;
+}
+
+/* Media query css for desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    background: #f4f5f7;
+  }
+
+  header {
+    padding: 24px 10%;
+  }
+
+  .skill-item img {
+    margin-left: 0;
+  }
+
+  #nav img {
+    display: none;
+  }
+
+  .nav-ul {
+    display: block;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    min-height: 100px;
+    width: 80%;
+    margin-left: 10%;
+    max-width: 1200px;
+  }
+
+  .section-intro {
+    padding-left: 15%;
+    padding-right: 15%;
+    background-size: cover;
+  }
```

Comment:
- [ ] Ouch! this is close but please make sure the background of the landing page is aligned properly as shown in the Figam design:
- Your design:
  ![image](https://user-images.githubusercontent.com/104968236/236420224-60102d60-35e8-4c47-b5ea-a0b18c66c81f.png)
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/236420329-e5e3e1c8-1625-4132-ad2e-0270486af14e.png)